e2e/api/steps.js

```diff
@@ -204,8 +203,8 @@ module.exports = {
 
     await assertSubmittedEvent('AWAITING_CASE_DETAILS_NOTIFICATION', {
       header: 'Notification of claim sent',
-      body: 'What happens next'
-    });
+      body: 'The defendant legal representative\'s organisation has been notified and granted access to this claim.'
+    }, true);
```

Comment:
Can we make the last argument `true` by default in `assertSubmittedEvent` method? There's only one event that doesn't have submitted callback